src/contexts/AuthProvider.js

```diff
@@ -0,0 +1,124 @@
+import { auth, db } from "../backend";
+import { useNavigate } from "react-router-dom";
+import { collection, doc, setDoc, getDocs, getDoc, serverTimestamp, } from "firebase/firestore";
+import { createUserWithEmailAndPassword, updateProfile, signOut, signInWithEmailAndPassword } from "firebase/auth";
+import { createContext, useContext, useState, useEffect } from "react";
+
+const AuthContext = createContext(null);
```

Comment:
try to stick to es6 syntax

Reply:
Hey @ravikumar0403 , I didn't get this. Can you be a little more specific?

Reply:
commented at the wrong line. what I meant though is use es6 arrow function syntax to create components

Reply:
Okay, Cool. Got it :+1: